contentcuration/contentcuration/frontend/shared/data/changes.js

```diff
@@ -182,3 +195,242 @@ export class ChangeTracker {
     });
   }
 }
+
+// These fields should not be included in change objects that we
+// store in the changes table for syncing to the backend
+// as they are only used for tracking state locally
+const ignoredSubFields = [COPYING_FLAG, LAST_FETCHED, TASK_ID];
+
+function omitIgnoredSubFields(obj) {
+  return omit(obj, ignoredSubFields);
+}
+
+class Change {
+  constructor({ type, key, table } = {}) {
```

Comment:
I tried this, and while webpack didn't complain when bundling, eslint gave a parsing error, so I'm leaving it as is for now.

Reply:
Hmm, it is slightly dangerous, because if you assign something like an object, that will be shared by all instances. With a string, it shouldn't be a big deal, but :+1: